lib/l10n/app_en.arb

```diff
@@ -1638,10 +1638,11 @@
   "@keepScreenOn": {
     "description": "Option to keep the screen on while using the app"
   },
-  "keepScreenOnSubtitle": "Keep the screen on while playing music or only while showing lyrics",
+  "keepScreenOnSubtitle": "Keep the screen on while using the app, while playing music or only while showing lyrics",
```

Comment:
You could shorten this to `When to keep the screen on` or something similar. The options are given in the select anyway ^^